src/github.com/getlantern/flashlight/config/flags.go

```diff
@@ -89,3 +100,14 @@ func (updated *Config) applyFlags() error {
 
 	return nil
 }
+
+func parseFrontFQDNs(frontFQDNs string) (map[string]string, error) {
+	fqdns := map[string]string{}
+	if err := yaml.Unmarshal([]byte(frontFQDNs), fqdns); err != nil {
+		return nil, err
+	}
+	if len(fqdns) == 0 {
```

Comment:
Actually, if you just take out the validation from here and leave the rest of the code as is (other than changing the comment in hostFn(), I think this will work nicely.